projects/imagemagick/rotate_fuzzer.cc

```diff
@@ -0,0 +1,23 @@
+#include <cstdint>
+
+#include <Magick++/Blob.h>
+#include <Magick++/Image.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  if (Size < 8) {
+    return 0;
+  }
+  double Degrees = *reinterpret_cast<const double *>(Data);
+  if (!isfinite(Degrees)) {
+    return 0;
+  }
+  const Magick::Blob blob(Data + 8, Size - 8);
```

Comment:
I think it's not a good idea to assume that `double` is always 8 bytes, would be better to use `sizeof(Degrees)` instead.

Reply:
Or maybe consider a hash-based approach explained in one of the comments above. What sort of data are we processing here? Should it be a valid image file?